hazelcast/config.py

```diff
@@ -531,8 +531,8 @@ def __init__(self):
         self._retry_initial_backoff = 1.0
         self._retry_max_backoff = 30.0
         self._retry_jitter = 0.0
-        self._retry_multiplier = 1.0
-        self._cluster_connect_timeout = 120.0
+        self._retry_multiplier = 1.05
+        self._cluster_connect_timeout = -1
```

Comment:
Can we move `-1` to a global variable, something like `DEFAULT_CLUSTER_CONNECT_TIMEOUT` and replace instances of it?